go/olc.go

```diff
@@ -142,11 +145,10 @@ func CheckShort(code string) error {
 // CheckFull checks the code whether it is a valid full code.
 // If it is short, it returns ErrShort.
 func CheckFull(code string) error {
-	if err := Check(code); err != nil {
```

Comment:
Why is the call to Check() being removed?

Reply:
CheckShort already includes Check.

Reply:
Got it, thanks for the clarification.